code/datums/mutations/active.dm

```diff
@@ -0,0 +1,48 @@
+/datum/mutation/human/adrenaline_rush
+	name = "Adrenaline Rush"
+	desc = "Allows the host to trigger their body's adrenaline response at will."
+	quality = POSITIVE
+	text_gain_indication = span_notice("You feel pumped up!")
+	instability = 15
+	power_path = /datum/action/cooldown/adrenaline
+
+	energy_coeff = 1
+	synchronizer_coeff = 1
+	power_coeff = 1
+
+/datum/mutation/human/adrenaline_rush/modify()
+	. = ..()
+	var/datum/action/cooldown/adrenaline/to_modify = .
+	if(!istype(to_modify)) // null or invalid
+		return
+
+	to_modify.adrenaline_amount = 10 * GET_MUTATION_POWER(src)
+	to_modify.comedown_amount = 7 / GET_MUTATION_SYNCHRONIZER(src)
+
+/datum/action/cooldown/adrenaline
+	name = "Adrenaline!"
+	desc = "Energize yourself, pushing your body to its limits!"
+	button_icon = 'icons/mob/actions/actions_genetic.dmi'
+	button_icon_state = "adrenaline"
+
+	cooldown_time = 2 MINUTES
+	/// How many units of each positive reagent injected during adrenaline.
+	var/adrenaline_amount = 10
+	/// How many units of each negative reagent injected after comedown.
+	var/comedown_amount = 7
+
+/datum/action/cooldown/adrenaline/Activate(mob/living/carbon/cast_on)
+	if(cast_on.stat)
+		return
```

Comment:
Should be in `isAvailable`, but also just use the check flag `AB_CHECK_CONSCIOUS`

Reply:
isavailable doesnt have a mob argument nor does the type have it as a variable